.github/ISSUE_TEMPLATE/feature.yml

```
@@ -10,28 +10,26 @@ body:
         You'll need to demonstrate widespread support for your idea among the community.
 
         Major feature proposals should generally be discussed on [Discourse](https://discuss.python.org/c/ideas/6) before opening a GitHub issue. Wait until it's clear that most people support your idea before filling in this form.
-  - type: checkboxes
+  - type: dropdown
     attributes:
       label: Has this already been discussed elsewhere?        
```

Remove trailing whitespace:
```suggestion
      label: Has this already been discussed elsewhere?
```